src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php

```diff
@@ -1018,4 +1034,18 @@ protected function get_menu_id( $menu_item_id ) {
 
 		return $menu_id;
 	}
+
+	/**
+	 * Prime menu item objects.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param WP_Post[] $posts Array of posts
+	 *
+	 * @return WP_Post[]
```

Comment:
```suggestion
	 * @return WP_Post[] Unmodified array of posts.
```

Possibly note in the docblock that this is intended to run on the `posts_results` filter and therefore needs to return an unmodified value.